src/Components/ToDo/ToDo.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './ToDo.scss';
+import User from '../User/User';
+
+function ToDo(prop) {
```

Comment:
```suggestion
function ToDo({ task }) {
```
Делай деструктуризацию  props